int32 mode
int32 ref_vel
int32 servo
int32 esc
int32 current
int32 lf
int32 rf
int32 lr
int32 rr
int32 motor
int32 kalman_lf
int32 kalman_rf
int32 kalman_lr
int32 kalman_rr
int32 kalman_gy
int32 kalman_ax
int32 gx
int32 gy
int32 gz
int32 mx
int32 my
int32 mz
int32 ax
int32 ay
int32 az
